ToDoApi copy/src/main/java/com/theironyard/Entities/User.java

```diff
@@ -0,0 +1,151 @@
+package com.theironyard.Entities;
+
+import org.hibernate.annotations.ColumnDefault;
+
+import javax.persistence.*;
+import java.math.BigInteger;
+import java.security.SecureRandom;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+
+/**
+ * Created by darionmoore on 1/9/17.
+ */
+@Entity
+@Table(name = "users")
+public class User {
+    private static final int EXPIRATION_DAYS = 1;
+
+    @Id
+    @GeneratedValue
+    private int id;
+
+    @Column(nullable = false)
+    private String userName;
+
+    @Column(nullable = false)
+    private String password;
+
+    @Column(nullable = false)
+    private String email;
+
+    @Column(nullable = false)
+    private String firstName;
+
+    @Column(nullable = false)
+    private String lastName;
+
+    @Column(nullable = true, unique = true)
+    private String token;
+
+    @Column(nullable = false)
+    @ColumnDefault("'2017-01-10'")
+    private LocalDateTime expiration;
+
+
+
+    public User() {
+        setTokenAndExpiration();
+
+    }
+
+    public User(String userName, String password) {
+        this.userName = userName;
+        this.password = password;
+    }
+
+    public User(int id, String userName, String password, String email, String firstName, String lastName) {
```

Comment:
Don't put in the ids for the entities.  You won't have one if you create it and if hibernate creates it it will use the default constructor